qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -219,7 +221,10 @@ def setText(self, text):
         if not text:
             pass
         elif text[0] in modeparsers.STARTCHARS:
-            super().set_prompt(text[0])
+            if config.val.completion.exit_prefix:
+                super().set_prompt('')
```

Comment:
This doesn't work correctly - for example, pressing `<Home>` now goes to the beginning of the line. Instead, you can probably remove `miscwidgets._CommandValidator`.